cmd/upload/manifest.go

```diff
@@ -45,16 +54,32 @@ var ManifestCmd = &cobra.Command{
 		}
 
 		manifestId := int32(i)
-		req := api.UploadManifestRequest{ManifestId: manifestId}
-		port := viper.GetString("agent.port")
-		conn, err := grpc.Dial(":"+port, grpc.WithTransportCredentials(insecure.NewCredentials()))
+		workflowPath, err := cmd.Flags().GetString("workflow")
+		workflowOpts, err := cmd.Flags().GetString("workflowOpts")
+
 		if err != nil {
-			fmt.Println("Error connecting to GRPC Server: ", err)
-			return
+			fmt.Println("Workflow error: ", err)
+		}
+		WrkFlwReq := api.StartWorkflowRequest{
+			ManifestId:   manifestId,
+			WorkflowFlag: workflowPath,
 		}
+		if workflowPath != "" {
+			fmt.Println(workflowOpts)
+			workflowResponse, err := client.StartWorkflow(context.Background(), &WrkFlwReq)
```

Comment:
probably worth checking for the above error as well.

Reply:
Included a check for this err on line 74